bin/run

```diff
@@ -113,17 +106,78 @@ else:
             np.random.standard_normal(spec['shape']).tolist()
         )
 
-with tfe.Session() as sess:
-    print("initing!!!")
-    sess.run(tf.global_variables_initializer())
-
-    print("running")
-    feed_dict = {
-        pl: input_data[i] for i, pl in enumerate(pls)
-    }
-    output = sess.run(
-        y.reveal(),
-        feed_dict=feed_dict,
-        tag='prediction'
-    )
-    print(output)
+
+runs = config.epochs
+if config.plaintext:
+    print('running prediction in plaintext')
+    with tfe.Session() as sess:
+        sess.graph.as_default()
+        tf.import_graph_def(graph_def, name='')
+        sess.run(tf.global_variables_initializer())
+
+        # TODO This isn't a good way to get the input/output nodes and we shouldn't
+        #      hardcode the input either!
+        print('running', runs, 'predictions')
+        for x in range(0, runs + 1): # first run is warmup
+            output = sess.graph.get_operations()[-2].values()[0]
+            input = sess.graph.get_operations()[0].values()[0]
+
+            sess.run(output, {input: np.random.rand(1000,528)}, tag='prediction')
```

Comment:
we're loading the input data from the input_file cli argument, should be able to pass input_data directly in here